src/app/helptext/vm/devices/device-add-edit.ts

```diff
@@ -85,17 +85,17 @@ rootpwd_tooltip : T('Enter a password for the <i>rancher</i> user. This\
 boot_placeholder : T('boot'),
 boot_tooltip : '',
 
-vnc_port_placeholder : T('Port'),
-vnc_port_tooltip : T('Can be set to <i>0</i>, left empty for FreeNAS to\
+port_placeholder : T('Port'),
+port_tooltip : T('Can be set to <i>0</i>, left empty for FreeNAS to\
```

Comment:
Just happened to watch some PRs. Shouldn't this be `TrueNAS` instead of `FreeNAS`?

Reply:
Yes, you are correct. Thanks for the review! Will update.